apps/website/docs/api/primitives/query.md

```diff
@@ -12,6 +12,7 @@ const query: Query<Params, Data, Error, InitialData>; // InitialData is allowed
 query.$data; // Store<Data | InitialData>
 query.$error; // Store<Error | null>
 query.$status; // Store<'initial' | 'pending' | 'done' | 'fail'>
+query.$idle; // Store<boolean>
```

Comment:
🤓 let's add a note about first appearance? Like `// Store<boolean>, since v0.8.0`

Reply:
Added for a query and mutation :)